portal/src/main/scala/cromwell/pipeline/controller/UserController.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.server.Directives.{complete, get, onComplete, parameter, path}
+import cromwell.pipeline.datastorage.dto.user.UserResponse
+import play.api.libs.json.Json
+import scala.concurrent.Future
+import scala.util.Failure
+import cromwell.pipeline.service.{UserService}
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import scala.concurrent.ExecutionContext
+import scala.util.Success
+
+class UserController (userService: UserService)(implicit executionContext: ExecutionContext) {
+  val route: Route =
+    path("users") {
+      get {
+        parameter('email.as[String]) { email =>
+          val res: Future[Seq[UserResponse]] = userService.getUsersByEmail(email)
+          onComplete(res) {
+            case Success(r)                            => complete(getSeqJson(r))
+            case Failure(exception)                    => complete(StatusCodes.InternalServerError)
+          }
+        }
+      }
+    }
+
+  private def getSeqJson(seq: Seq[UserResponse]): String = {
+    Json.stringify(Json.toJson(seq))
+  }
+}
```

Comment:
Turn on ensure line feed option in the Intellij Idea to get rid of this git warning.

Reply:
done